Only in include/: Makefile
--- varnish-6.0.0/include/vpf.h
+++ include/vpf.h
@@ -30,6 +30,8 @@
 #ifndef VPF_H_INCLUDED
 #define VPF_H_INCLUDED

+#include <sys/stat.h>
+
 struct vpf_fh;

 struct vpf_fh *VPF_Open(const char *path, mode_t mode, pid_t *pidptr);
--- varnish-6.0.0/include/vsb.h
+++ include/vsb.h
@@ -30,7 +30,7 @@

 #ifndef VSB_H_INCLUDED
 #define VSB_H_INCLUDED
-
+#include <sys/types.h>
 /*
  * Structure definition
  */